File: neat.py
import sys
import select
from comic.comic import * 

foo = Comic()

for i in range(60):
    foo.tick(1,0,0,0,1,0)

for i in range(20):
    for j in range(2):
        foo.tick(0,0,0,0,1,0)
    for j in range(5):
        foo.tick(1,0,0,0,1,0)

    

File: neat.jl
include("./comic.jl") # make module comic "visible"
import .Comic
# change working directory to comic folder for the library to not go haywire
# it is important to reset the working directory to the comic subdirectory if changed
cd(string(pwd(), "/comic"))

# some basic movement
Comic.add_instance(1,0,0,1)

for i in 1:10
    Comic.tick(1)
end
